src/utils.ts

```diff
@@ -1,19 +1,9 @@
 import { ObjectId } from 'mongodb';
```

Comment:
Those change are hard to follow, I wonder if we can add some basic tests using `ts-expect`?

Reply:
Which changes exactly? All of these new types added here are using inside `mongodbTypes`, which has extensive type tests. But I can write some more tests for these helpers.

Reply:
I've added tests for the new types I added in this file.